Description: Fix typo in smtpd.conf.5
Author: Ryan Kavanagh <rak@debian.org>
Origin: Debian
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/smtpd/smtpd.conf.5
===================================================================
--- opensmtpd.orig/smtpd/smtpd.conf.5	2015-10-06 16:09:26.511746831 -0400
+++ opensmtpd/smtpd/smtpd.conf.5	2015-11-01 15:18:22.357855428 -0500
@@ -564,7 +564,7 @@
 .Ic ciphers Ar cipher-list
 .Xc
 Specify an alternate ciphers list to use when establishing TLS sessions.
-It is highly recommanded to avoid making use of this option unless there
+It is highly recommended to avoid making use of this option unless there
 is a good understanding of the implications.
 .Pp
 When not specified, only ciphers considered safe are chosen.
